src/ast/mod.rs

```diff
@@ -1,3 +1,5 @@
+// SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
I am not sure what this copyright line means -- if it was an intentional inclusion, can you please explain the rationale for it and if it was not intentional remove it

Reply:
This was an intentional inclusion, primarily marking recognition of ownership for code, due to my affiliation with the organization I work for (Nvidia). 

If this open source project doesn't use/accept such copyright licenses for pr's I'm happy to go ahead and remove it.

Reply:
> If this open source project doesn't use/accept such copyright licenses for pr's I'm happy to go ahead and remove it.

Please remove it. The code in this repository is licensed with the Apache Software License 2.0 and by contributing you are also agreeing to license your contribution under the same terms. 

https://github.com/sqlparser-rs/sqlparser-rs/blob/main/LICENSE.TXT

If this is unacceptable we will not be able to accept this contribution

Reply:
I am sorry if this was not clear -- I have added some clarification in the main project README: https://github.com/sqlparser-rs/sqlparser-rs/pull/570

Reply:
Thanks for the clarification. This makes sense to me, I'll go ahead and remove the copyright license from these files.